src/System.Private.CoreLib/shared/System.Private.CoreLib.Shared.projitems

```diff
@@ -294,7 +294,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)System\Number.Formatting.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Number.Grisu3.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Number.NumberBuffer.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)System\Number.NumberToDouble.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\Number.NumberToFloatingPointBitsRoslyn.cs" />
```

Comment:
This should have more self-describing name than "Roslyn".

Reply:
`NumberToFloatingPointBits` is probably the most self-describing name. I can remove the `Roslyn` moniker from the end.

Reply:
Fixed.

Reply:
Did you mean to also rename `NumberToFloatingPointBitsRoslyn` method?

Reply:
Yes, will fix.